### PR TITLE
ci: split tests

### PR DESCRIPTION
**Pull request by idoshamun**
Use CircleCI to run tests in parallel to reduce the CI time

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]
Does it also tackle the issue with rerunning only failed tests?
(Seeing you already use circleci runner)

File: .circleci/config.yml [hunk 1]
```diff
@@ -34,6 +34,7 @@ jobs:
           - POSTGRES_DB: api_test
           - POSTGRES_PASSWORD: 12345
       - image: redis:6
+    parallelism: 4
```
Comment (capJavert) [inline .circleci/config.yml:37]: Does this also create 4 databases for each run?

Comment (idoshamun) [reply .circleci/config.yml:37]: Yes. Basically it runs the same job 4 times with the full config and just split the tests using some circleci utils

**Comment by idoshamun** [comment]
@rebelchris not sure tbh, haven't tried

File: .circleci/config.yml (hunk 1 above)
Comment (capJavert) [reply .circleci/config.yml:37]: Perfect!

**Pull request merged**